README.md

```diff
@@ -428,6 +428,53 @@ When the playbook is done running, if you got no errors you can login at:
     username: admin
     password: admin
 
+
+Enabling the firmware upgrader module
+-------------------------------------
+
+To enable the firmware upgrader module you need to set `openwisp2_firmware_upgrader` to `true` in
+your `playbook.yml` file. Here's a short summary of how to do this:
+
+**Step 1**: [Install ansible](#install-ansible)
+
+**Step 2**: [Install this role](#install-this-role)
+
+**Step 3**: [Create inventory file](#create-inventory-file)
+
+**Step 4**: Create a playbook file with following contents:
+
+```yaml
+- hosts: openwisp2
+  become: "{{ become | default('yes') }}"
+  roles:
+    - openwisp.openwisp2
+  vars:
+    openwisp2_firmware_upgrader: true
+```
+
+**Step 5**: [Run the playbook](#run-the-playbook)
+
+When the playbook is done running, if you got no errors you can login at:
+
+    https://openwisp2.mydomain.com/admin
+    username: admin
+    password: admin
+
+**Note**: You can configure `openwisp-firmware-upgrader specific settings <https://github.com/openwisp/openwisp-firmware-upgrader#settings>`_
+using `openwisp2_extra_django_settings` variable of this ansible role.
+For example if you want to enable the `APIs of openwisp-firmware-upgrader <https://github.com/openwisp/openwisp-firmware-upgrader#rest-api>`_,
```

Comment:
I would switch to giving the example to disable the API, as we discussed I will make the API enabled by default in openwisp-firmware-upgrader: https://github.com/openwisp/openwisp-firmware-upgrader/issues/111.